packages/gatsby/src/utils/get-latest-apis.ts

```diff
@@ -18,7 +18,7 @@ export const getLatestAPIs = async (): Promise<IAPIResponse> => {
 
     return data
   } catch (e) {
-    if (await fs.existsSync(OUTPUT_FILE)) {
+    if (await fs.pathExistsSync(OUTPUT_FILE)) {
```

Comment:
https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/pathExists-sync.md

Reply:
How come? I don't see the benefit, but happy to be convinced!

Reply:
I'm sorry I didn't explain it enough. I encountered following errors on CircleCI: 
`TypeError: _fsExtra.default.existsSync is not a function`
https://circleci.com/gh/gatsbyjs/gatsby/345789?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
`TypeError: _fsExtra.default.pathExistsSync is not a function`
https://circleci.com/gh/gatsbyjs/gatsby/345965?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
Can you find why this error occurs?